### PR TITLE
[runtime] use libtorch 1.9.0 by default

### PR DESCRIPTION
**Pull request by robin1001**

### CODE REVIEW

File: runtime/server/x86/CMakeLists.txt [hunk 1]
```diff
@@ -52,9 +52,9 @@ FetchContent_Declare(boost
...
 include_directories(${boost_SOURCE_DIR})
 
-# third_party: libtorch 1.6.0, use FetchContent_Declare to download, and
+# third_party: libtorch 1.9.0, use FetchContent_Declare to download, and
 # use find_package to find since libtorch is not a standard cmake project
-set(PYTORCH_VERSION "1.6.0")
+set(PYTORCH_VERSION "1.9.0")
 if(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
     set(LIBTORCH_URL "https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-${PYTORCH_VERSION}%2Bcpu.zip")
     set(URL_HASH "SHA256=dacb2af654f1cffa43b0f3187ce945e1c93305b4c8bb60361c7a8cb543e1da82")
```
Comment (xingchensong) [inline runtime/server/x86/CMakeLists.txt:60]: update URL_HASH for 1.9.0 ?

**Review by pengzhendong — APPROVED** [review]

**Review by xingchensong — COMMENTED** [review]
do we need to update URL_HASH for 1.9.0?

```sh
curl https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-1.6.0%2Bcpu.zip | sha256sum
# dacb2af654f1cffa43b0f3187ce945e1c93305b4c8bb60361c7a8cb543e1da82

curl https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-1.9.0%2Bcpu.zip | sha256sum
# 9bcd5b2703e18f2374413e55907ac20b2e5d8f5ae06a669f2e9905096114b4f1
```

File: runtime/server/x86/CMakeLists.txt (hunk 1 above)
Comment (robin1001) [reply runtime/server/x86/CMakeLists.txt]: fixed

**Review by pengzhendong — APPROVED** [review]

**Comment by gstswwx** [comment]
@robin1001
I think the pytorch version should be updated to 1.9.0 in `wenet/runtime/device/android/wenet/app/build.gradle`.

https://github.com/wenet-e2e/wenet/blob/5ef4974463300866e1b6db01058fd5da5a14bdf1/runtime/device/android/wenet/app/build.gradle#L78-L79

**Pull request merged**